javascript/ecmascript-2015/symbol/symbol.md

```diff
@@ -44,45 +44,45 @@ During creation you can also set a description for the symbol which is useful ma
 
 ```javascript
 var sym = Symbol("Enki");
-console.log(sym)
+console.log(sym);
 // Symbol(Enki)
 ```
 
 Note that supplying a description when creating a symbol does not add the symbol to the symbol registry which we will discuss shortly.
 
-The typeof method of course returns symbol:
+The `typeof` method, of course, returns `"symbol"`:
 
 ```javascript
 typeof Symbol(); //"symbol"
 ```
 
-When you create a symbol it is unique even if you create symbols with the same names:
+When you create a symbol, it is unique even if you create symbols with the same names:
```

Comment:
So here we refer to the parameter passed into `Symbol` as "name" but above we refer to it as "description".

["description" is the correct term](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol#Parameters)

We should change this sentence to something like:

> When you create a symbol, it is unique even if there are other symbols with the same description

Reply:
done